pkg/versionstream/versionstreamrepo/gitrepo_integration_test.go

```diff
@@ -0,0 +1,70 @@
+// +build integration
+
+package versionstreamrepo_test
+
+import (
+	"testing"
+
+	"github.com/jenkins-x/jx/pkg/gits"
+	"github.com/jenkins-x/jx/pkg/versionstream/versionstreamrepo"
+	"github.com/stretchr/testify/assert"
+)
+
+const (
+	RepoURL    = "https://github.com/jenkins-x/jenkins-x-versions"
```

Comment:
I would prefer to see these tests executed against something other than the real versions repo, as I worry that this approach makes them quite fragile. We normally use `test_data` to validate these sorts of methods, which I think will be a more robust approach and what we should do here. This will require you to make a git repo of that test data to start with.

Reply:
Will do.